content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -163,6 +163,17 @@ When reviewing, use the following as a starting point.
 - Do the changes show up in the Netlify preview? Be particularly vigilant about lists, code
   blocks, tables, notes and images.
 
+### Blog
+
+- Early feedback of blog posts are welcome via a Google Doc or HackMD. Please request for early feedback on the [#sig-docs-blog Slack channel](https://kubernetes.slack.com/archives/CJDHVD54J).
+- Before reviewing blog PRs, be familiar with the [Submitting blog posts and case studies](/docs/contribute/new-content/blogs-case-studies/).
+- Blog posts from kubernetes.dev can be mirrored on kubernetes.io within 5 days of initial publication.
```

Comment:
"within 5 days"? What does this mean?

Reply:
Could we use hours instead? In general, it's more universal and not timezone dependent.

Reply:
This is meant to clarify any unwritten rules (if any) on mirroring blog posts from kubernetes.dev to kubernetes.io

Reply:
The time frame is removed from the PR
Marking resolved